zombies.js

```diff
@@ -79,6 +87,22 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+  this._pack = [];
+  this.getPack = function() {
+    return this._pack;
+  };
+  this._maxHealth = health;
+  this.getMaxHealth = function() {
+    return this._maxHealth;
+  };
```

Comment:
`_pack` and `_maxHealth` should be put on a const so that they are truly private. Update the references to `_pack` and `_maxHealth` in your `getMaxHealth` and your `getPack` functions.